scout/adapter/mongo/variant_events.py

```diff
@@ -468,6 +468,8 @@ def unmark_partial_causative(self, institute, case, user, link, variant):
 
         # update partial_causative field of this case
         partial_causatives = case.get("partial_causatives") or {}
+        if variant["_id"] not in partial_causatives:
```

Comment:
Not that I disagree really, but any particular reason why this would happen in regular usage? Some warning, log or error might be in order? Silent fails can be a bit scary.

Reply:
This would not be useful on  a regular base, but I can imagine (and I have tested) this case:
having 2 tabs open :
1. Removing the partial causative from the case page
1. After point 1, trying to remove the partial causative also from the other tab (variant tab).

I could add a log, but I don't really think it will be very useful, because the effect of the cosde is that the page would refresh and the partial causative would disappear, as if the command works.
Does it make any sense?

Reply:
You are probably right: sounds good, and I have no situation where I think it would be dangerous. Lets use this!

Reply:
Ok, but I'll merge then, thanks for the review!